lib/elixir/lib/record.ex

```diff
@@ -293,21 +297,27 @@ defmodule Record do
   # Normalizes of record fields to have default values.
   @doc false
   def __fields__(type, fields) do
-    :lists.map(fn
-      {key, value} when is_atom(key) ->
-        try do
-          Macro.escape(value)
-        rescue
-          e in [ArgumentError] ->
-            raise ArgumentError, "invalid value for record field #{key}, " <> Exception.message(e)
-        else
-          value -> {key, value}
-        end
-      key when is_atom(key) ->
-        {key, nil}
-      other ->
-        raise ArgumentError, "#{type} fields must be atoms, got: #{inspect other}"
-    end, fields)
+    :lists.map(
+      fn
+        {key, value} when is_atom(key) ->
+          try do
+            Macro.escape(value)
+          rescue
+            e in [ArgumentError] ->
+              raise ArgumentError,
+                    "invalid value for record field #{key}, " <> Exception.message(e)
+          else
+            value -> {key, value}
+          end
+
+        key when is_atom(key) ->
+          {key, nil}
+
+        other ->
+          raise ArgumentError, "#{type} fields must be atoms, got: #{inspect(other)}"
+      end,
```

Comment:
Please extract this function to a variable and call `:lists.map(..., fields)` on one line :).

Reply:
Done